packages/firestore/src/local/indexeddb_mutation_queue.ts

```diff
@@ -598,6 +578,58 @@ export class IndexedDbMutationQueue implements MutationQueue {
   }
 }
 
+/**
+ * Delete a mutation batch and the associated document mutations.
+ * @return done, A PersistencePromise that signals when the deletion has completed, and removedDocuments, the document
+ *         mutations that were removed.
+ */
+export function removeMutationBatch(
+  txn: SimpleDbTransaction,
+  userId: string,
+  batch: MutationBatch
+): { removedDocuments: DocumentKey[]; done: PersistencePromise<void> } {
```

Comment:
Should this match the order in return? `done` is before `removedDocuments` in return.

Reply:
This has been refactored away, but in general I don't think it matters. Objects are unordered.